package.json

```diff
@@ -129,7 +129,7 @@
     "css-loader": "0.28.0",
     "devtron": "^1.4.0",
     "electron": "^1.6.11",
-    "electron-builder": "^19.7.1",
+    "electron-builder": "19.43.0",
```

Comment:
this actually breaks the build 😢 

i'll try a smaller upgrade path via one of the builds referenced here: https://github.com/develar/electron-builder/commit/d56326b6bd9b5f76457a63923150029b6b58cc37